src/main/java/core/exception/ReadException.java

```diff
@@ -0,0 +1,7 @@
+package core.exception;
+
+public class ReadException extends RuntimeException {
+    public ReadException(String message) {
+        super(message);
```

Comment:
why do you create only a read exception? Read what? Rename it 
```suggestion
public class FileReadException extends RuntimeException {
    public FileReadException(String message) {
        super(message);
```